### PR TITLE
Added Go library

### PR DESCRIPTION
**Pull request by tmornini**

### CODE REVIEW

File: dist/README.md [hunk 1]
```diff
@@ -106,14 +106,15 @@ Links
...
 Server Support
-- Ruby _(brotli)_ - https://github.com/miyucy/brotli
-- Node _(brotli)_ - https://www.npmjs.com/package/brotli
-- Node _(iltrob)_ - https://github.com/MayhemYDG/iltorb
-- Koa _(koa-send)_ - https://github.com/koajs/send
-- Express _(shrink-ray)_ - https://github.com/aickin/shrink-ray
-- NGINX _(ngx_brotli)_ - https://github.com/google/ngx_brotli
 - Apache _(mod_brotli)_ - https://httpd.apache.org/docs/trunk/mod/mod_brotli.html
+- Express _(shrink-ray)_ - https://github.com/aickin/shrink-ray
+- Go _(brotli)_ - https://github.com/dsnet/compress/tree/master/brotli
```
Comment (snuggs) [inline dist/README.md]: @tmornini Should we use https://github.com/dsnet/compress ?

Comment (tmornini) [reply dist/README.md]: We should, and now we do. 👍

**Comment by tmornini** [comment]
@snuggs Can't until the travis build passes...

**Comment by snuggs** [comment]
@tmornini ah i see. This is a PERFECT example of what we spoke about earlier in the year.

`MutationObserver` has been in Flux for a bit and the web platform tests and therefore jsdom are as well. I'll try to see what I can do in the meantime in our project. But the fix should be upstreamed to Jsdom and then finally web-platform-tests.

**Pull request merged**